src/plugins/console/public/application/components/console_menu.tsx

```diff
@@ -76,13 +76,10 @@ export class ConsoleMenu extends Component<Props, State> {
     }
   }
 
-  copyText(text: string) {
-    const textField = document.createElement('textarea');
-    textField.innerText = text;
-    document.body.appendChild(textField);
-    textField.select();
-    document.execCommand('copy');
-    textField.remove();
+  async copyText(text: string) {
+    if (window.navigator?.clipboard) {
+      await window.navigator.clipboard.writeText(text);
+    }
```

Comment:
Will this drop support in what version, would be great to add a comment.

Reply:
According to [caniuse](https://caniuse.com/?search=clipboard), `clipboard` is supported by the browsers updated at least in the passed 3 years. Barring IE, all those that supported `execCommand` do support `clipboard`.